packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -454,7 +454,7 @@ async def read(self) -> Optional[bytes]:
             if not data:  # pragma: no cover
                 return None
             return data
-        except asyncio.streams.IncompleteReadError as e:
+        except asyncio.streams.IncompleteReadError as e:  # pragma: no cover
```

Comment:
on my local machine, this is already covered by  `tests/test_packages/test_connections/test_p2p_libp2p/test_errors.py::TestP2PLibp2pConnectionFailureNodeDisconnect::test_node_disconnect`
(https://github.com/fetchai/agents-aea/blob/develop/tests/test_packages/test_connections/test_p2p_libp2p/test_errors.py#L143)